docs/notes.rst

```diff
@@ -5,9 +5,14 @@ This release includes the following features:
 
 * Partial implementation of allocation QoS
 * Implementation of Discovery Server
+* Modified default behavior for unknown writers.
+* Added timeout when sending through UDP
+* Asynchronous thread with non-blocking API
```

Comment:
The two above should be combined into a single one: "Implementation of non-blocking calls"